mobile/library/objective-c/BUILD

```diff
@@ -49,6 +47,8 @@ objc_library(
     }),
     visibility = ["//visibility:public"],
     deps = [
+        ":envoy_aliases_lib",
```

Comment:
Doesn't this dep need to go in `envoy_objc_bridge_lib` so that the header is available when building it?

Reply:
it's a part of `envoy_objc_bridge_lib ` already - but you are right, it's redundant here I think. Removing